Report.md

```diff
@@ -1,69 +1,89 @@
+  - [Introduction](#introduction)
+  - [Methodology](#methodology)
+  - [Analysis and Findings](#analysis-and-findings)
+    - [Temperature](#temperature)
+    - [Humidity](#humidity)
+    - [Precipitation](#precipitation)
+  - [Discussion](#discussion)
+  - [Conclusion](#conclusion)
 
 
-# **Introduction:**
+## Introduction
 
 Climate science has definitively proven that global temperatures have drastically increased relative to past decades. Higher global temperatures have implications for many climatological phenomena, and increase the frequency and desctructiveness of natural disasters like storms, floods, droughts, and wildfires. In my hometown of Washington D.C., I've always known hot summer seasons, however in the past ten years I've noticed that all our seasons have felt extraordinarily warmer. Summer storms have also become more intense, with seemingly heavier rains and more frequent flooding. This project seeks to explore some the impacts of climate change; specifically how fluctuations in temperature impact humidity, precipitation, and the relationship between those two variables. 
 
 [Research](https://www.metoffice.gov.uk/about-us/news-and-media/media-centre/weather-and-climate-news/2020/scientists-investigate-humidity---the-second-pillar-of-climate-change) has shown how rising temperatures influence shifts in the saturation of water vapor in the air, which in turn increases the liklihood of heavier raindfall incidents. I hypothesize that my analysis will show first that temperatures have increased increased in recent years, leading to higher humidity readings and increased precipitation.
 
 Here is the [link to the Github repository](https://github.com/griffinberonio/Climate_Data_Project) for this project.
 
-# **Methodology**
+## Methodology
 
 This study examines ten-year periods from five total weather stations across two separate regions: the Midwest and the East Coast. Weather data comes from weather stations at five major airports: Asheville Regional Airport (ARA) in Ashevill North Carolina, Reagan National (DCA) in Washington D.C. JFK Airport (JFK) in New York City, Minneapolis-Saint Paul (MSP) in Minneapolis, Minnesota, and Dane County Regional (MSN) in Madison, Wisconsin. I first examined fluctuations in temperature averages and maximums using wetbulb temperature data for each station. I also compared temperature trends between the two regions, and analyzed departures from normal temperatures. Next, I looked at fluctuations in humidity between locations and regions, using daily humidity averages from the data. Finally, I looked at precipitation totals, averages, and maximums across time and locations, and examined relationships between humidity and rainfall. 
 
-# **Analysis and Findings**
+## Analysis and Findings
+
+### Temperature
 
-## *Temperature*
 
-* *Temperature Trends by Location*
-![image](Images/location_temp_plot.png)
+| ![image](Images/location_temp_plot.png) |
+|:--:|
+| Temperature Averages by Location |
```

Comment:
Tables are a good way to get captions. I couldn't find a better way in Markdown. 

Google docs makes you do it this way too.